src/lib/index.js

```diff
@@ -481,13 +481,6 @@ lib.containsAny = function(s, fragments) {
     return false;
 };
 
-// get the parent Plotly plot of any element. Whoo jquery-free tree climbing!
-lib.getPlotDiv = function(el) {
```

Comment:
Beautiful commit. 

Note that `Lib.isPlotDiv` is currently used in W2. No such problem for `Lib.getPlotDiv`.